spec/inputs/base/validations_spec.rb

```diff
@@ -338,5 +338,145 @@ class MyInput
       end
     end
   end
+
+  describe '#validation_min' do
+    before :example do
+      allow(instance).to receive(:validations?).and_return(:true)
+      allow(instance).to receive(:validations).and_return([validator])
+    end
+
+    context 'with a greater_than numericality validator' do
+      let(:validator) { double(options: { greater_than: option_value }, kind: :numericality) }
```

Comment:
Would it be too hard to use real instance of `ActiveModel::Validations::NumericalityValidator` ?